src/components/MovieList/MovieList.jsx

```diff
@@ -1,5 +1,12 @@
 import './MovieList.scss';
+import { MovieCard } from '../MovieCard';
 
-export const MovieList = () => (
-  <>Put the list here</>
+export const MovieList = ({ movies }) => (
+  <>
+    {movies.map(movie => (
+      <div className="card" data-cy="Movie" key={movie.imdbId}>
```

Comment:
The '.card' block should be extracted to a 'MovieCard' component. The current implementation repeats the 'card' div in the 'MovieList' component which may lead to unnecessary code duplication and possible inconsistencies.